blueflood-core/src/main/java/com/rackspacecloud/blueflood/service/LocatorFetchRunnable.java

```diff
@@ -189,7 +191,8 @@ public void executeRollupForLocator(RollupExecutionContext executionContext, Rol
         try {
             // get a list of all locators to rollup for a shard
             locators.addAll(IOContainer.fromConfig().getLocatorIO().getLocators(getShard()));
-        } catch (Exception e) {
+            locatorsPerShard.update(locators.size());
```

Comment:
it's already there but below the executionContext.markUnsuccessful(e); call.  Let me move it up.